src/shell/CMakeLists.txt

```diff
@@ -177,11 +177,10 @@ FOREACH(T ${LEANTESTS})
   endif()
 ENDFOREACH(T)
 
-# Create a lake test for each subdirectory of `lake/examples` which contains a `test.sh` file.
-file(GLOB_RECURSE LEANLAKETESTS "${LEAN_SOURCE_DIR}/lake/examples/test.sh")
+# Create a lake test for each subdirectory of `lake` which contains a `test.sh` file.
+file(GLOB_RECURSE LEANLAKETESTS "${LEAN_SOURCE_DIR}/lake/test.sh")
```

Comment:
@tydeu I noticed we didn't even run most of the Lake test suite, which would considerably weaken my claim that Lean CI might be sufficient for Lake :)